lib/apipie_bindings/api.rb

```diff
@@ -213,7 +213,7 @@ def http_call(http_method, path, params={}, headers={}, options={})
       if dry_run?
         empty_response = ApipieBindings::Example.new('', '', '', 200, '')
         ex = options[:fake_response ] || empty_response
-        response = create_fake_response(ex.status, ex.response, http_method, path, args)
+        response = create_fake_response(ex.status, ex.response, http_method, URI.join(@uri, path).to_s, args)
```

Comment:
In previous versions of the bindings @uri was not mandatory when running against cache and not live API. Though I can see reasons of making it mandatory even for dry_runs I'd keep this change conditional for cases @uri is defined.
This is currently breaking many hammer tests.